app/src/main/java/org/mozilla/fenix/home/sessioncontrol/viewholders/pocket/PocketStoriesComposables.kt

```diff
@@ -0,0 +1,283 @@
+            start = annotationStartIndex,
+            end = annotationEndIndex
+        )
+    }
+
+    Column(
+        modifier = Modifier.padding(vertical = 16.dp),
+        horizontalAlignment = Alignment.CenterHorizontally
+    ) {
+        content()
+
+        // Don't yet have the bottom image from designs as a proper Android SVG.
+        Box(
+            Modifier
+                .background(Color.Red)
+                .size(64.dp, 27.dp)
+                .padding(top = 16.dp)
+        )
+        // Image(
+        //     painterResource(R.drawable.ic_firefox_pocket),
+        //     "Firefox and Pocket logos",
+        //     Modifier
+        //         .size(64.dp, 27.dp)
+        //         .padding(top = 16.dp),
+        //     contentScale = ContentScale.FillHeight
+        // )
+
+        CompositionLocalProvider(LocalContentAlpha provides ContentAlpha.medium) {
+            ClickableText(
+                text = annotatedText,
+                style = MaterialTheme.typography.caption,
+                onClick = {
+                    annotatedText
+                        .getStringAnnotations("link", it, it)
+                        .firstOrNull()?.let {
+                            println("Learn more clicked! Should now access ${it.item}")
+                        }
+                }
+            )
+        }
+    }
+}
+
+/**
+ * Displays [content] in an expandable card.
+ */
+@Composable
+fun expandableCard(content: @Composable (() -> Unit)) {
+    var isExpanded by remember { mutableStateOf(true) }
+    val chevronRotationState by animateFloatAsState(targetValue = if (isExpanded) 0f else 180f)
+
+    Card(
+        modifier = Modifier.fillMaxWidth(),
+        shape = RoundedCornerShape(4.dp),
+        onClick = { isExpanded = !isExpanded }
+    ) {
+        Column {
+            Row(
+                modifier = Modifier.fillMaxWidth(),
+                verticalAlignment = Alignment.CenterVertically
+            ) {
+                Text(
+                    modifier = Modifier.weight(10f),
+                    text = "Trending stories from Pocket",
+                    style = MaterialTheme.typography.h6,
+                    maxLines = 1,
+                    overflow = TextOverflow.Ellipsis
+                )
+
+                IconButton(
+                    onClick = { isExpanded = !isExpanded },
+                    modifier = Modifier.rotate(chevronRotationState)
+                ) {
+                    Icon(
+                        modifier = Modifier.weight(1f),
+                        painter = painterResource(id = R.drawable.ic_chevron_up),
+                        contentDescription = "Expand or collapse Pocket recommended stories",
+                    )
+                }
+            }
+
+            AnimatedVisibility(visible = isExpanded) {
+                content()
+            }
+        }
+    }
+}
+
+@Composable
+@Preview
+private fun finalDesign() {
```

Comment:
I'd say all Composables should try to preview the end result.
The auto-update is great to get a feel of changes in any child Composables without the need to run the app on a device.
To get past the "non-default parameters" errors when Compose does not know what to render we can use `PreviewParameterProvider` or wrap the Composables in another Composable like here from where we can try to provide values for all parameters.